docs/sphinx/user/inputs_SyntheticTurbulence.rst

```diff
@@ -0,0 +1,77 @@
+Section: SyntheticTurbulence
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section is for setting turbulence injection parameters.
+
+.. input_param:: SynthTurb.turbulence_file
+
+   **type:** String, required
+   
+   Name of the netcdf file that contains the data.
+   
+.. input_param:: SynthTurb.wind_direction
+
+   **type:** Real, optional, default = 270.
+   
+   The wind direction. 
+   
+.. input_param:: SynthTurb.grid_location 
+
+   **type:** List of Reals, required
+  
+   Location of the turbulent grid.
+
+.. input_param:: SynthTurb.mean_wind_type
+
+   **type:** String, required 
+  
+   The type of profile to use. Options are ConstValue, LinearProfile, and PowerLawProfile.
+
+.. input_param:: SynthTurb.grid_spacing 
+
+   **type:** Real, optional, default = 2.0
+  
+   The grid spacing used in the box.
+
+.. input_param:: SynthTurb.time_offset
+
+   **type:** Real, optional, default = 0.0
+  
+   The time offset betweent he data and the simulation.
+
+.. input_param:: SynthTurb.zref
+
+   **type:** Real, required
+  
+   The default reference height at the center of the turbulence grid
```

Comment:
The docs should clarify that zref, uref, shear_exponent, zoffset, umin, and umax are only used when the mean wind type is the Power Law Profile.

Reply:
Agree, just added them under the mean_wind_type section.